### ISSUE TITLE
Blender white screen frozen

### ISSUE THREAD

**Issue by kalkulatorr** [jamesstringerparsec/Easy-GPU-PV#410]

**Comment by timminator** [comment]
That the drivers are not the same is correct.
The problem is that Blender uses OpenGL and OpenGL is not really supported.
One thing I noticed is that you are using an AMD graphics card and the latest drivers seem to have problems with GPU PV as mentioned in this [thread](https://github.com/jamesstringerparsec/Easy-GPU-PV/issues/392).
But I dont think that will help.
There's also this [thread](https://github.com/jamesstringerparsec/Easy-GPU-PV/issues/342) about enabling OpenGL with a few things that you could try out but it could be that there's currently no way of getting Blender to run using GPU PV.

**Comment by kalkulatorr** [comment]
yeah, i tried all of that before, still nothing works, probably as you say OpenGL is no supported yet.
